components/Card.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { ReactComponent as Checked } from '../assets/images/checked.svg'
+
+type Props = {
+  children?: React.ReactNode
+  success?: boolean
+  text?: string
+  classes?: string
+  title: string
+}
+
+const Card: React.FC<Props> = props => {
+  const classes = `col-sm-${props.classes ||
+    'col-sm-8 col-md-7 col-lg-6 col-xl-4 m-auto'}`
```

Comment:
so if `props.classes` is null, then `classes` variable is:

`col-sm-col-sm8` <- what is that?